main.py

```diff
@@ -26,13 +27,21 @@
 
 date_today = date.today()
 
-sell_today, predictions = get_predictions(MODEL, TRAINING_DAYS, PREDICTED_DAYS, VALUE, UNITS, COST)
+sell_today, predictions = get_predictions(
+    MODEL, TRAINING_DAYS, PREDICTED_DAYS, VALUE, UNITS, COST
+)
+
+# Generate a plot and table for the server
+plot_to_file("projection", predictions, sell_today, VALUE)
+
+# Still need a line to include this on the server!
```

Comment:
This shouldn't be necessary as the server runs a bash script that runs main.py every hour.

Reply:
not sure i understand, this is the line that makes the graph that the server will show,. Unless you are querying the comment on line 37, that was just a comment i forgot to remove

Reply:
No worries, it was about the comment, could you remove it if you get a chance please?

Reply:
done